.github/CODEOWNERS

```diff
@@ -584,7 +584,7 @@
 /core/templates/pages/release-coordinator-page/ @oppia/release-workflow-reviewers
 /scripts/release_scripts/ @oppia/release-workflow-reviewers
 /assets/release_constants.json @oppia/release-workflow-reviewers
-
+/assets/release_constants_docker.json @oppia/release-workflow-reviewers
```

Comment:
Hi @gp201 why are we duplicating this? Now we have two sources of truth and there is nothing that keeps them the same.

This is not good. Can we avoid doing that?

Reply:
@seanlip, if I am right, your concern is making 2 files in sync, rather than having 2 files in the first place.

If yes, your concern is justified. In solution, we can add `override_docker` in `release_constants.json`, then we can use some shell script to create `release_constants_docker.json`. We can also add that same script to `make build`. So, everytime `make build` is run, we have new constants file for docker.

example: `release_constants.json`
```json
{
  "cons1": "val1",
  "cons2": "val2",
  "override_docker": {
    "cons2": "new_val2",
    "cons3": "val3_specifically_for_docker"
  }
}
```
will generate following `release_constants_doceker.json`
```json
{
  "cons1": "val1",
  "cons2": "new_val2",
  "cons3": "val3_specifically_for_docker"
}
```

Reply:
Hm, keeping the 2 files in sync partially alleviates the problem, but my concern actually is having 2 files in the first place, because devs need to remember to make both of them -- unless one is procedurally generated.

But it would be better to use a single file throughout. Why can't we do that?

Reply:
I don't think we could use one file, as many constants for docker such as paths, are different in the Python environment. If we want to have one file, we can do it by adding more constants for docker in the same file, but then we will need to use `OPPIA_IS_DOCKERIZED` in many more places, which I don't think is a good idea.

We could replicate the Python environment in Docker, but that would add a huge work load and I am concerned about whether we would be able to use all features of Docker like modularization. 

Also, I couldn't understand "unless one is procedurally generated" in your previous comment. The solution I suggested [here](https://github.com/oppia/oppia/pull/19963#discussion_r1531459679) generates whole release_constants_docker.json procedurally using the script, thus devs need to maintain just release_constants.json and other will be generated automatically specifically for docker.

Reply:
@jayam04 Could you list which paths are different and why they are different? Maybe we need to rethink some of those constants.

Yes, the solution you suggested fits within the idea I suggested in my first paragraph. But I want to understand why exactly we can't have a single file before committing to that other approach.

Reply:
![image](https://github.com/oppia/oppia/assets/93824505/1b43488f-84d4-4047-920a-7addbcbd3059)
Currently, it's only `GCLOUD_PATH`

Reply:
OK, thanks @jayam04. I think the current solution is overkill. Just drop GCLOUD_PATH from release_constants.json for now, as well as the test that checks it. I will file an issue on the release-scripts repo to get GCLOUD_PATH from elsewhere, thanks.

Reply:
I removed the GCLOUD_PATH from release_constants.json and the tests.

Reply:
Thanks, do we still need this file here then? Can we drop it?

Reply:
Fixed